src/SprykerEco/Zed/AvalaraTax/Business/AvalaraTaxBusinessFactory.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AvalaraTax\Business;
+
+use Avalara\AvaTaxClient;
```

Comment:
* I think it's better to wrap `AvaTaxClient` and `TransactionBuilder` with an adapter.

Reply:
Resolved.